src/main/java/RacingCar.java

```diff
@@ -0,0 +1,113 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+import java.util.Scanner;
+
+public class RacingCar {
+
+    static String move = "-";
+
+    static List<Car> recordMovingCar = new ArrayList<>();
+
+    public static void main(String[] args) {
+
+        InputView userInput = scannerInput();
+
+        int numberOfCarsValue = userInput.getNumberOfCars();
+        int numberOfAttemptsValue = userInput.getNumberOfAttempts();
```

Comment:
```suggestion
        int numberOfCars = userInput.getNumberOfCars();
        int numberOfAttempts = userInput.getNumberOfAttempts();
```

- 네이밍시에 의미를 가지지 않거나, 중복되는 부분은 최대한 없애보세요